lib/datasource/remote/api/network_repository.dart

```diff
@@ -22,6 +23,7 @@ abstract class NetworkRepository {
   String tableUsers = 'users';
   String tableVoice = 'voice';
   String tableVotes = 'votes';
+  String tableVoters = 'voters';
```

Comment:
could name these "referendumVoters" and proposalVotes for clarity

Reply:
Overall, I don't think having these constants makes much sense, unless they're used in more than 1 place. 

They're all in different contracts... so if anything the definition of a location of a table would have to have both the contract name and the table name, then it would make more sense.\

Just e generic comment, nothing to do with this PR ;)